### DISCUSSION TITLE
Feature Request: Issues & PRs - Can we have private or individual labels?

### DISCUSSION THREAD

**Discussion by kingthorin** [community/community#14474] [Projects and Issues]
I'd really like to be able to label PRs and Issues with labels that are specifically for me and only relevant to me.

Use cases:
- I want to track my thoughts on effort related to particular issues.
- I want to track stalled PRs that I might try to help get over the finish line.
- I want to track issues or PRs that I should mention when I do Sponsor updates (via GitHub sponsors).
- etc (I'm sure the community can and would come up with 5 tons of uses if this were available).

None of which is relevant to the project(s) I participate in or teams I need to interact with.

Edit: A slight variation brought up b y @gparant https://github.com/github/feedback/discussions/11651 labels that are only visible to GitHub teams not necessarily end users.

**Comment by kingthorin** [comment]
Anyone else have use cases that could benefit from this?

**Reply by dwmarshall** [reply]
My use case (in GitHub Enterprise) is that I look at a big list of pull requests that need review every day. Sadly, some of these pull requests are long-lived. If I decide I don't have anything to contribute to a pull request, I'd like to put some label on it that reminds me I don't have anything to add. No one else would be interested in my decision not to approve or comment on such a pull request.

**Reply by kingthorin** [reply]
That's a great idea, I can see that being handy for sure.

**Comment by kingthorin** [comment]
Now that github.community has been migrated to discussions, I thought I'd bump this and see if anyone has further use case ideas.

**Comment by ringods** [comment]
Another usecase.

When a company builds software in the open, all GH issues are public as well. That's OK. 

Now it would be handy if there was the option of private labels. Using (org wide) private labels, issues could be tagged with private customer-related labels, as a way to see which customer is impacted by a certain public GH issue.

**Reply by tillprochaska** [reply]
I have a similar use case: I work on open-source software that has lots of non-technical users. We get feature requests and bug reports from these users through other channels, and it would be nice to be able to track which users reported a bug/requested a feature to be able to reach out to them once the bug is fixed/we decided to start working on a feature. Right now, I maintain a separate document that references issue numbers and users/organizations.

**Reply by kingthorin** [reply]
Great use case.

**Comment by chrisgaun** [comment]
I would love to see this in the new Github Projects. Right now you can choose between "text, single select" etc. The problem with free text is everyone may add what is supposed to be standardized text differently. With single select the text can be standardized but they can only choose one. Labels would solve both of these issues and are easy to search.

**Reply by justinretzolk** [reply]
Adding to this, it would solve an issue that we have for our Project's main View.

Ideally, our "main" View would be anything with the `prioritized` label, however, this:

1. Isn't cross-repository
2. Doesn't work for Draft items

The first one could be solved by adding the label to any repository whose issues/PRs wind up on the Project, but some of those aren't directly in our teams jurisdiction (if you will), so it would be a bit odd to do so (not to mention if the label happened to be used by any other teams for different reasons in those repositories). The second isn't something there's currently a workaround for.

**Comment by kingthorin** [comment]
Bump 😁
